spec/quby/answers/services/score_calculator_spec.rb

```diff
@@ -51,6 +51,52 @@ module Quby::Answers::Services
       end
     end
 
+    describe '#ensure_answer_values_for', focus: true do
```

Comment:
`focus: true` vergeten weg te halen.

In RoQua hebben we een failsafe waardoor circle faalt als er ergens nog een focus staat, zie https://github.com/roqua/roqua/blob/master/spec/spec_helper.rb#L181 . Lijkt me aardig om die ook voor quby toe te voegen.